g2p/static/custom.js

```diff
@@ -223,7 +226,7 @@ function createTable(index, data) {
     let headerLabels = headers.map(header => header.replace('_', ' ').split(' ').map(x => { return x.charAt(0).toUpperCase() + x.slice(1) }).join(' '))
     var hotSettings = {
         data: data,
-        columns: headers.map(x => { return { 'data': x, type: 'text' } }),
+        columns: headers.map(x => { if (x !== "prevent_feeding") { return { 'data': x, type: 'text' } } else { return { 'data': x, type: 'checkbox' } } }),
```

Comment:
I'm used to our black formatting in Python converting such lines to multi-line. Are there similar conventions for JS or is that more free-for-all?

JS has so many nastily nested braces and parens, I would not want something anywhere as string as black for it, but I guess intuitively I have have formatted this code like this, although I'm not really satisfied with this option either: 

```
columns: headers.map(x => { if (x !== "prevent_feeding") { 
             return { 'data': x, type: 'text' }
         } else {
             return { 'data': x, type: 'checkbox' }
         }}),
```

Don't take this as a real suggestion, I don't actually like it! I'm just discussing this as part of my learning JS.